### PR TITLE
Reader Full Post block: add action links

### LINKED ISSUE #7815: Reader Refresh: add edit button to full post view
post-action-links

### PR DESCRIPTION
**Pull request by bluefuton**
Add action links to bottom of full post: edit, share, comment and like.

ex-actions

Fixes #7815 and #7816.

Test live: https://calypso.live/?branch=add/reader/full-post-block-action-links

### CODE REVIEW

**Comment by bluefuton** [comment]
- [ ] Fix scroll-to-top when clicking Share

**Comment by bluefuton** [comment]
@blowery is tackling the scroll-to-top glitch in a separate PR. It only happens for users with > 6 sites.

**Comment by bluefuton** [comment]
Scroll-to-top glitch merged in https://github.com/Automattic/wp-calypso/pull/7883. Just rebased this PR on top of it.

File: client/blocks/post-edit-button/index.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	return (
+		<a className="post-edit-button" href={ editUrl }>
+			<Gridicon icon="pencil" size={ size } className="post-edit-button__icon" />
+			<span className="post-edit-button__label">{ translate( 'Edit' ) }</span>
+		</a>
+	);
+};
+
+PostEditButton.propTypes = {
+	post: React.PropTypes.object.isRequired,
```
Comment (blowery) [inline client/blocks/post-edit-button/index.jsx]: what would you think about calling these `postId` and `siteId` so as to make it clear that they're just IDs and not the full post / site objects?

Comment (blowery) [inline client/blocks/post-edit-button/index.jsx]: oh, they _are_ full post and site objects. nevermind

Comment (bluefuton) [inline client/blocks/post-edit-button/index.jsx]: In this case, they're the full `post` and `site` objects.

`getEditURL` requires them: https://github.com/Automattic/wp-calypso/blob/67618156a019f547a3ed164f767b8dbd2b2725f5/client/lib/posts/test/utils.js#L20

File: client/blocks/post-edit-button/index.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import { localize } from 'i18n-calypso';
+import { getEditURL } from 'lib/posts/utils';
+
+const PostEditButton = ( { post, site, size, translate } ) => {
+	const editUrl = getEditURL( post, site );
+	return (
+		<a className="post-edit-button" href={ editUrl }>
```
Comment (blowery) [inline client/blocks/post-edit-button/index.jsx]: should probably track clicks on this. are we doing that for the edit link in the menu?

Comment (bluefuton) [inline client/blocks/post-edit-button/index.jsx]: That's a good shout. We're firing `calypso_reader_edit_post_clicked` in the post options menu, so we can do the same here.

We're already tracking the Share, Comment and Like buttons.

**Comment by bluefuton** [comment]
- [ ] Add tracking to post-edit-button

File: client/blocks/post-edit-button/index.jsx (hunk 2 above)
Comment (bluefuton) [inline client/blocks/post-edit-button/index.jsx]: Added in aca11af.

**Comment by fraying** [comment]
LGTM! Nice work!

**Comment by blowery** [comment]
Ship it, assuming the conflicts resolve cleanly

**Comment by bluefuton** [comment]
Rebasing now 👍

**Pull request merged**